.dockerignore

```diff
@@ -1,7 +1,20 @@
-docker
-thinkhazard/static/build
-thinkhazard/static/font
-thinkhazard/locale/**/*.mo
-thinkhazard/locale/**/*.po
-thinkhazard.egg-info
-Dockerfile
```

Comment:
Looks like it was shorter with blacklist than with whitelist ?

Reply:
Yes, but black list was not exhaustive (.covegare, .nodeids, eclipse files, ...).

Until now I had to permanently keep some custom files and folders here, for example "tmp" that I mount as volumes for caches and raise errors on build (owner postgres).

And I do not want to put custom .mk files with access keys and password in docker images that I could publish on docker hub in case of manual push.

Now everybody should build approximately the same locally as on CI.

Reply:
=> Better reproduce what happens on CI